app/components/Header.tsx

```diff
@@ -0,0 +1,176 @@
+"use client";
+import { Link } from "@chakra-ui/next-js";
+import { Box, Grid, Stack } from "@chakra-ui/react";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+
+export default function Header() {
+  const [hamburger, setHamburger] = useState(false);
+
+  const pathname = usePathname();
+
+  const handleHamburger = () => {
+    setHamburger(!hamburger);
+  };
+
+  return (
+    <Stack>
+      <Grid templateColumns={"500px 1fr 100px"}>
+        <Box p="12px">
+          <Image
+            src="/images/kenworthy-logo.jpg"
+            alt="Kenworthy Machine"
+            width={500}
+            height={500}
+          />
+        </Box>
+        <Box
+          display="flex"
+          alignItems="center"
+          justifyContent="flex-end"
+          gap="24px"
+        >
+          <Link
+            href="/"
+            color="white"
+            fontSize="24px"
+            borderBottom={
+              pathname === "/" ? "solid var(--primary-red) 2px" : "none"
+            }
+            _hover={{
+              color: "var(--text-color-light)",
+              borderBottom: "solid var(--primary-red) 2px",
+              transitionDuration: "400ms",
```

Comment:
My comment for lines 106-116 can be applied here as well if we decide to go that route